src/driver.rs

```diff
@@ -277,7 +312,15 @@ pub fn main() {
         };
 
         let mut no_deps = false;
-        let clippy_args = env::var("CLIPPY_ARGS")
+        let clippy_args_var = env::var("CLIPPY_ARGS").map_or_else(
+            |e| match e {
+                VarError::NotPresent => None,
+                VarError::NotUnicode(s) => panic!("CLIPPY_ARGS is not valid Unicode: {:?}", s),
```

Comment:
We shouldn't panic for error handling. I don't think non-unicode clippy args should be a fatal error. You can just print a message, that the arguments are ignored and move on.

Reply:
> You can just print a message, that the arguments are ignored and move on.

Then I can't figure out what should be set as the value of environment variable in `env_depinfo` if the argument is not Unicode. If it's set to empty or `None`, will Cargo rebuild every time even the non-Unicode value is not changed?

Reply:
If there is a non-unicode character in the arguments, you can just set the `CLIPPY_ARGS` to `None` and treat it like they aren't present at all. As long as some non-unicode char is present cargo shouldn't rebuild everything, since it stays `None`.